File: Documentation/4.0/Samples/nodejs/client-api/creatingDocumentStore.js
import { DocumentStore } from "ravendb";

//region document_store_creation
store = new DocumentStore(["http://localhost:8080"], "Northwind");
store.initialize();
//endregion

//region document_store_holder
// documentStoreHolder.js
const store = new DocumentStore("http://localhost:8080", "Northwind");
store.initialize();
export { store as documentStore }; 
//endregion

File: Documentation/5.2/Samples/java/src/test/java/net/ravendb/ClientApi/CreatingDocumentStore.java
package net.ravendb.ClientApi;

import net.ravendb.client.documents.DocumentStore;
import net.ravendb.client.documents.IDocumentStore;

public class CreatingDocumentStore {
    public CreatingDocumentStore() {
        //region document_store_creation
        try (IDocumentStore store = new DocumentStore( new String[]{ "http://localhost:8080" }, "Northwind")) {
            store.initialize();


        }
        //endregion
    }



    //region document_store_holder
    public static class DocumentStoreHolder {

        private static IDocumentStore store;

        static {
            store = new DocumentStore(new String[]{ "http://localhost:8080" }, "Northwind");
        }

        public static IDocumentStore getStore() {
            return store;
        }
    }
    //endregion
}
